### ISSUE TITLE
Add class with custom name

### ISSUE THREAD

**Issue by mhsjlw** [dgrunwald/rust-cpython#88]
Currently, you must create a class like:

```rust
py_class!(class PyWorld |py| {
  ...
}
```

But if I'd like to make the class `World` rather than `PyWorld` inside of the python program, how can I `add_class` in the module initializer with the name as `World` rather than `PyWorld`. I have read [the documentation](https://dgrunwald.github.io/rust-cpython/doc/cpython/struct.PyModule.html#method.add_class) and it doesn't seem like it's possible. Any ideas?

**Comment by dgrunwald** [comment]
Agreed, that would be a nice feature since it's often necessary to avoid naming collisions when wrapping existing Rust code for Python.

Maybe `py_class!(class World as PyWorld |py| ...)`?

Possible workaround:

```
mod my_python_classes {
   py_class!(pub class World |py| {
    ...
  }
}
use my_python_classes::World as PyWorld;
```
